src/module/item/SR5Item.ts

```diff
@@ -1610,7 +1618,7 @@ export class SR5Item extends Item {
      * Configure the given matrix item to be controlled by this item in a PAN/WAN.
      * @param target The matrix item to be connected.
      */
-    async addNetworkDevice(target: SR5Item) {
+    async addNetworkDevice(target: SR5Item|SR5Actor) {
```

Comment:
Not necessary as a change, but I try (and sometimes forget) to instead of using these local union types to define a new type like so
```TypeScript
export type NetworkDeviceDocuments = SR5Item|SR5Actor 
```

This way, I can see where this specific combination of Item and Actor is used for this specific purpose and can change it easily across the project, if needed. :)

Reply:
Good idea!